requirements.txt

```diff
@@ -16,3 +16,4 @@ black
 pytest
 coverage
 pytest-cov
+pygrib~=2.1.4
```

Comment:
Not familiar with the versions of this package. Is it safe to do `~=` rather than `==`?

Reply:
`~=` on a patch version _shouldn't_ cause any problems. The Pipfile just specifies `pygrib = "*"` so we're being more cautious here already.

Reply:
I think `~=` is a good middle ground - safer than `*` and less work to maintain than `==`. That's what i'd use by default, and only use `==` for special cases or unstable packages.